sunpy/util/cond_dispatch.py

```diff
@@ -78,7 +78,7 @@
 
 from itertools import chain, repeat
 
-from sunpy.extern.six.moves import zip as izip
+from sunpy.extern.six.moves import zip
```

Comment:
My thoughts were that it would be better to leave the normal names in the code and just modify it at import time. For a start people are likely to forget to use `six.map` everywhere.